e2e/template_e2e_tests/run_e2e_test.sh

```diff
@@ -61,27 +61,29 @@ done
 # The following vars need to be set in the environment when runnin the e2e tests.
 # For example, set up env vars and action secrets in Github Action workflow.
 echo "ORGANIZATION_ID=$ORGANIZATION_ID"
-echo "FOLDER_ID=$FOLDER_ID"
 echo "BILLING_ACCOUNT=$BILLING_ACCOUNT"
+echo "FOLDER_ID=$FOLDER_ID"
 echo "GOOGLE_APPLICATION_CREDENTIALS=$GOOGLE_APPLICATION_CREDENTIALS"
 
 # Parsing arguments
-skip_cleanup="" is_create_new_project=""
-while getopts "n" flag
+skip_cleanup=""
+is_create_new_project=""
```

Comment:
It is safer to have an explicit value for `is_create_new_project` rather than a non-empty string